### DISCUSSION TITLE
Detecting Full Node vs Miner on the network.

### DISCUSSION THREAD

**Discussion by qlpqlp** [dogecoin/dogecoin#2717] [Help needed] [answered]
Hello to all Dev's :)

Is there a way we can detect on the network a Minner?

In outher words, the peers connected to a Node, does it indicates if its a Node or a Miner?

Thanks in advance.

**Comment by rnicoll** [comment, marked as answer]
A miner requires a full node (ish... there's ways to hack it up without a full node, but... they're extremely ill-advised). There definitely shouldn't be (it would be an operational risk if someone could identify miners, as they could be then be DDoSed to halt the network).

There are behaviours which would give away a likely light node, in that they only request/relay headers and not full blocks, but it's not a clean boolean, it's observing what the node does and taking a guess.

**Reply by qlpqlp** [reply]
Thanks :)

My objective was to count how many miners are in reality and not just full nodes :)

Lets keep it this way lol :)
